charts/redpanda/templates/tests/test-kafka-internal-tls-status.yaml

```diff
@@ -59,26 +59,10 @@ spec:
         do sleep 2
         done
       resources: {{ toYaml .Values.statefulset.resources | nindent 12 }}
-      volumeMounts:
-        - name: {{ template "redpanda.fullname" . }}
-          mountPath: /tmp/base-config
-        - name: config
-          mountPath: /etc/redpanda
+      volumeMounts: {{ include "default-mounts" . | nindent 8 }}
   {{- range $name, $cert := .Values.tls.certs }}
         - name: redpanda-{{ $name }}-cert
           mountPath: {{ printf "/etc/tls/certs/%s" $name }}
   {{- end }}
```

Comment:
Why this is left behind?

Reply:
I see it being defined in https://github.com/redpanda-data/helm-charts/pull/733/files#diff-8407f32f870c88088a3041f8a5a8394806973dabdef154b7fde83d5e77f70ed8R697-R702

Reply:
I just really enjoy making CI fail over and over while I go through and find all the mistakes I made. :facepalm: